package.json

```diff
@@ -31,6 +32,7 @@
     "vue": "^3.2.38"
   },
   "dependencies": {
-    "vue": "^3.2.38"
+    "vue": "^3.2.38",
+    "vue-select": "^4.0.0-beta.5"
```

Comment:
Why is this a different version number than the version installed under devDependencies?